FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/Models/FhirToDataLakeOrchestratorJobResult.cs

```diff
@@ -30,6 +30,16 @@ public class FhirToDataLakeOrchestratorJobResult
         /// </summary>
         public long CompletedJobCount { get; set; }
 
+        /// <summary>
+        /// Processing jobs in the process of submitting, used for system scope
+        /// </summary>
+        public string[] SubmittingProcessingJob { get; set; } = null;
+
+        /// <summary>
+        /// Next job timestamp to be processed for each resource type, used for system scope
+        /// </summary>
+        public Dictionary<string, DateTimeOffset> SubmittedResourceTimestamps { get; set; } = new Dictionary<string, DateTimeOffset>();
```

Comment:
the comment is inconsistent with the variable name.

Reply:
Updated it, thanks.